### PR TITLE
ENG-968: Rework AWSCLIv2 unzip with 7za

### PR DESCRIPTION
**Pull request by Sudokamikaze**
Workers on rel started to stuck on command:
```
++ curl https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -o /tmp/awscliv2.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 33.7M  100 33.7M    0     0  99.2M      0 --:--:-- --:--:-- --:--:-- 99.2M
++ unzip /tmp/awscliv2.zip -d /tmp
Archive:  /tmp/awscliv2.zip
```
It appears sometimes and stops pipelines.
Also, there wasn't such issue on initial testing

### CODE REVIEW

File: IaC/ps.cd/init.groovy.d/cloud-ps3.groovy [hunk 1]
```diff
@@ -112,19 +112,23 @@ initMap['docker'] = '''
...
         echo try again
     done
 
-    unzip /tmp/awscliv2.zip -d /tmp
+    7za -o/tmp x /tmp/awscliv2.zip 
     cd /tmp/aws && sudo ./install
 
     sudo install -o $(id -u -n) -g $(id -g -n) -d /mnt/jenkins
 
+    sudo install -o $(id -u -n) -g $(id -g -n) -d /mnt/jenkins
```
Comment (delgod) [inline IaC/ps.cd/init.groovy.d/cloud-ps3.groovy:130]: why we need this line?

**Review by delgod — CHANGES_REQUESTED** [review]

File: IaC/cloud.cd/init.groovy.d/cloud.groovy [hunk 2]
```diff
@@ -94,16 +94,18 @@ initMap['docker'] = '''
...
+    sudo yum -y install java-1.8.0-openjdk git docker p7zip
+    sudo yum -y remove java-1.7.0-openjdk
+
     until curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "/tmp/awscliv2.zip"; do
         sleep 1
         echo try again
     done
 
-    unzip /tmp/awscliv2.zip -d /tmp
+    7za -o/tmp x /tmp/awscliv2.zip
```
Comment (EvgeniyPatlan) [inline IaC/cloud.cd/init.groovy.d/cloud.groovy:106]: why do we need p7zip? According to official doc unzip should work
https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: IaC/cloud.cd/init.groovy.d/cloud.groovy (hunk 2 above)
Comment (Sudokamikaze) [reply IaC/cloud.cd/init.groovy.d/cloud.groovy:106]: Yep, but as we discussed on DS, few runners stuck on `Archive: ` stage, although instance is not removed from AWS
And it started to stuck after `unzip` command

**Comment by Sudokamikaze** [comment]
Closing as it needs proper investigation and rework, pointed not to latest AWS Cli but to specific version on rel.cd(where it appeared) and re-checked other jenkins for this issue

**Pull request closed without merging**